### PR TITLE
docs: polish up the words in "record.rst"

### PR DESCRIPTION
**Pull request by BeEdward**
Fixed the wording of the documentation. It should explains it better and remove the label context of the document, since it is not truly relavent.

### CODE REVIEW

File: docs/source/schema/complex_types/record.rst [hunk 1]
```diff
@@ -2,11 +2,10 @@
  record
 ########
 
-The ``record`` type represents a labeled array capable of holding any data type, just like the
-struct in C++ or the Series in pandas.
+The ``record`` type is where we can definte complext data stuctures by grouping related variables togather in the same place. It is simiilar to
```
Comment (linjiX) [inline docs/source/schema/complex_types/record.rst]: `complext` -> `complex`
`togather` -> `together`
`simiilar` -> `similar`

File: docs/source/schema/complex_types/record.rst [hunk 2]
```diff
@@ -2,11 +2,10 @@
  record
 ########
 
-The ``record`` type represents a labeled array capable of holding any data type, just like the
-struct in C++ or the Series in pandas.
+The ``record`` type is where we can definte complext data stuctures by grouping related variables togather in the same place. It is simiilar to 
+``struct`` in C++ or the ``Series`` in pandas. It is preferred to use `record` to hold the grouped data in each row, and a column, in portex, is a series of records of the same type.
```
Comment (linjiX) [inline docs/source/schema/complex_types/record.rst]: please make the commit message follow the [angular commit message format](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#-commit-message-format)

Also make it more detailed

**Review by linjiX — CHANGES_REQUESTED** [review]

File: docs/source/schema/complex_types/record.rst (hunk 1 above)
Comment (linjiX) [reply docs/source/schema/complex_types/record.rst]: Portex doc use the third person. shall we remove the `we` in the doc?

Comment (linjiX) [reply docs/source/schema/complex_types/record.rst]: `definte` -> `define`
`stuctures` -> `structures`

Comment (linjiX) [reply docs/source/schema/complex_types/record.rst:5]: `stuctures` -> `structures`

**Review by linjiX — CHANGES_REQUESTED** [review]

File: docs/source/schema/complex_types/record.rst (hunk 2 above)
Comment (linjiX) [reply docs/source/schema/complex_types/record.rst]:
```
`record` -> ``record``
 use ``xxx`` syntax in rst file for code samples
```

**Review by linjiX — CHANGES_REQUESTED** [review]

File: docs/source/schema/complex_types/record.rst (hunk 1 above)
Comment (BeEdward) [reply docs/source/schema/complex_types/record.rst]: Done

File: docs/source/schema/complex_types/record.rst (hunk 2 above)
Comment (BeEdward) [reply docs/source/schema/complex_types/record.rst]: Done. Could you give me an example on what is the preferred header for this commit? Thx

**Comment by BeEdward** [comment]
All comments have been addressed in the new commit.

File: docs/source/schema/complex_types/record.rst (hunk 2 above)
Comment (linjiX) [reply docs/source/schema/complex_types/record.rst]: For me, the message of this commit can be:
```
docs: polish up the words in "record.rst"
```

**Pull request merged**